pkgs/by-name/nf/nfd/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  boost, # probably needs to match the one from ndn-cxx
```

Comment:
We could add it to passthru and take it from there to make things easier.

Reply:
Done, also for ndn-tools